content/en/tracing/profiler/enabling/native.md

```diff
@@ -0,0 +1,158 @@
+- `--environment`, the [environment][4] name, for example, `production`.
+- `--service`, the [service][4] name, for example, `web-backend`. If this is not specified, the profiler will use the default value `myservice`.
+- `--vername`, the [version][4] of your application.
+
+**Note**: if you usually launch your application using shell builtins, for example:
+
+```shell
+exec myapp --arg1
+```
+
+then you must invoke `ddprof` with those builtins instead:
+
+```shell
+exec ddprof --service "<SERVICE_NAME>" --vername "<APPLICATION_VERSION"> myapp --arg1
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+4. A minute or two after starting your application, your profiles will show up on the [Datadog APM > Profiler page][4].
+
+## Configuration
+
+Configuration for the profiler can be set by commandline parameters, environment variables, or a combination of both. Whenever both are provided for a given setting, the commandline parameter is preferred.
+
+| Environment variable         | Longname    | Shortname | Default   | Description                                                                                                                    |
+|------------------------------|-------------|-----------|-----------|--------------------------------------------------------------------------------------------------------------------------------|
+| DD_ENV                       | environment | E         |           | The [environment][5] name, for example, `production`.                                                                          |
+| DD_SERVICE                   | service     | S         | myservice | The [service][5] name, for example, `web-backend`.                                                                             |
+| DD_VERSION                   | vername     | V         |           | The [version][5] of your service.                                                                                              |
+| DD_AGENT_HOST                | host        | H         | localhost | The hostname for the Datadog agent.                                                                                            |
+| DD_TRACE_AGENT_PORT          | port        | P         | 8126      | The Datadog agent listening port.                                                                                              |
+| DD_TRACE_AGENT_URL           | url         | U         |           | `https://<hostname>:<port>` overrides other agent host/port settings.                                                          |
+| DD_TAGS                      | tags        | T         |           | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas, such as: `layer:api,team:intake`. |
+| DD_PROFILING_NATIVENICE      | nice        | i         |           | Sets the nice level of the profiler without affecting the instrumented processes.                                              |
```

Comment:
Nit- The "missing" underscores after NATIVE seem slightly odd. (I know this isn't a docs issue.)

Reply:
This is a good point.  If we didn't change these parameters now, then after releasing we may be in a position where we have to support bad/confusing names for a long time.  We've gone through the list and standardized `_` tokenization as well as the name of the parameters.

Good callout.  It's always been slightly annoying, but always ignored.  Thank you